src/deploy.go

```diff
@@ -84,10 +97,16 @@ func getRef() string {
 	return string(out.Bytes())
 }
 
+/*
+* Guess the correct mime type for the file extension
+ */
 func guessContentType(file string) string {
 	return mime.TypeByExtension(filepath.Ext(file))
 }
 
+/*
+* Should this file compress? use the NO_GZIP list to check if the extension is available for compression
```

Comment:
You're just reiterating the name of the function and exactly what the code inside of it says.  You can assume anyone reading this file speaks Go (and this comment is a liability if the code changes).